Selenium.AutomatedTests/AutomationScenarioBuilder.cs

```diff
@@ -137,6 +159,9 @@ public AutomationScenarioBuilder WithStep(IStep step)
             return this;
         }
 
+        /// <summary>
+        /// clean at the de-construction of the object 
```

Comment:
This is not needed. Picks summary from IDisposable.Dispose()

Reply:
It is solved in the commit 7f2820da5bb38f631308af8866df04ad9f81d2aa